lib/public/views/Logs/Details/LogTreeViewModel.js

```diff
@@ -118,7 +143,7 @@ export class LogTreeViewModel extends Observable {
      * Retrieve the tree of a given log from the API
      *
      * @param {number} selectedLogId The id of the log for which tree must be retrieved
-     * @returns {undefined} Injects the data object with the response data
+     * @return {void} Injects the data object with the response data
```

Comment:
JSDOC recommendation is to use `returns` rather than `return`
https://jsdoc.app/tags-returns.html 
Please update accordingly on the other changes you have made